### PR TITLE
upgrade: aspell from 0.60.8 to 0.60.8.1

### PR DESCRIPTION
**Pull request by SeanDougherty**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Upgraded from 0.60.8 to 0.60.8.1

###### Does this affect the toolchain?  

NO

###### Test Methodology

- local build
- BuddyBuild [[Link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=499196&view=results)]

### CODE REVIEW

**Review by neha170 — COMMENTED** [review]
Changes look good. Please fill out the description and build status, thanks

**Comment by SeanDougherty** [comment]
> Changes look good. Please fill out the description and build status, thanks

Thank you @neha170 , your feedback has been incorporated!

File: SPECS/aspell/aspell.spec [hunk 1]
```diff
@@ -15,8 +15,6 @@ URL:            http://aspell.net/
 Source:         ftp://ftp.gnu.org/gnu/aspell/aspell-%{version}.tar.gz
```
Comment (SeanDougherty) [inline SPECS/aspell/aspell.spec:15]: Need to update to: https://ftp.gnu.org/gnu/aspell/aspell-0.60.8.1.tar.gz format

File: cgmanifest.json [hunk 2]
```diff
@@ -595,8 +595,8 @@
         "type": "other",
         "other": {
           "name": "aspell",
-          "version": "0.60.8",
-          "downloadUrl": "ftp://ftp.gnu.org/gnu/aspell/aspell-0.60.8.tar.gz"
+          "version": "0.60.8.1",
+          "downloadUrl": "ftp://ftp.gnu.org/gnu/aspell/aspell-0.60.8.1.tar.gz"
```
Comment (SeanDougherty) [inline cgmanifest.json:599]: need to update to https://ftp.gnu.org/gnu/aspell/aspell-0.60.8.1.tar.gz format

File: SPECS/aspell/aspell.spec [hunk 3]
```diff
@@ -47,8 +45,6 @@ and header files needed for Aspell development.
 %setup -q
 %patch0 -p1 -b .fc
```
Comment (SeanDougherty) [inline SPECS/aspell/aspell.spec:46]: Need to update patch format

Comment (SeanDougherty) [reply SPECS/aspell/aspell.spec:46]: to patch # instead of patch#

**Comment by SeanDougherty** [comment]
> Changes look good. Please fill out the description and build status, thanks

Thank you @neha170 ! The PR is ready for your follow-up review when you have the chance 👍

File: SPECS/aspell/aspell.spec [hunk 4]
```diff
@@ -12,11 +12,9 @@ License:        LGPLv2+ AND LGPLv2 AND GPLv2+ AND BSD
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            http://aspell.net/
-Source:         ftp://ftp.gnu.org/gnu/aspell/aspell-%{version}.tar.gz
+Source:         https://ftp.gnu.org/gnu/aspell/aspell-%{version}.tar.gz
 Patch0:         aspell-0.60.7-fileconflict.patch
 Patch1:         aspell-0.60.7-pspell_conf.patch
-Patch2:         aspell-0.60.7-mp.patch
-Patch3:         CVE-2019-25051.patch
```
Comment (neha170) [inline SPECS/aspell/aspell.spec:19]: Looks like you forgot to remove this file

**Review by neha170 — APPROVED** [review]
Looks good. Please remove the CVE file before merging, thanks

**Pull request merged**